UNRELEASED.md

```diff
@@ -1,6 +1,16 @@
 # Unreleased
 
 > Place your changes below this line.
+**Breaking**
```

Comment:
Unless this has breaking changes to our consumers and how the components work, we can probably make this a patch. As breaking for us means it requires action on the consumer or visually breaking 😄

Reply:
Ahh got you. I thought breaking would apply because they were major dep upgrade :-)